scipy/sparse/linalg/dsolve/linsolve.py

```diff
@@ -383,3 +384,125 @@ def solve(b):
         return solve
     else:
         return splu(A).solve
+
+
+def spsolve_triangular(A, b, lower=True, overwrite_b=False):
+    """
+    Solve the equation `A x = b` for `x`, assuming a is a triangular matrix.
+    Parameters
+    ----------
+    A : (M, M) sparse matrix
+        A sparse square triangular matrix. Should be in CSR format.
+    b : (M,) or (M, N) array_like
+        Right-hand side matrix in `a x = b`
+    lower : bool, optional
+        Whether `A` is a lower or upper triangle matrix.
+        Default is lower triangle matrix.
+    overwrite_b : bool, optional
+        Allow overwriting data in `b`.
+        This may give a performance gain.
+    Returns
+    -------
+    x : (M,) or (M, N) ndarray
+        Solution to the system `A x = b`.  Shape of return matches shape of `b`.
+    Raises
+    ------
+    LinAlgError
+        If `A` is singular.
+    ValueError
+        If shape of `A` or shape of `b` do not match the requirements.
+    Notes
+    -----
+    .. versionadded:: 0.20.0
+    """
```

Comment:
This docstring is causing the TravisCI build to fail. Each of the underlined section headings should have a blank line before them.